crates/nu-command/tests/commands/def.rs

```diff
@@ -151,15 +151,39 @@ fn def_fails_with_invalid_name() {
 }
 
 #[test]
-fn def_errors_with_specified_list_type() {
+fn def_with_specified_list_type() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
         def test-command [ foo: list<any> ] {}
         "#
     ));
 
-    assert!(actual.err.contains("unknown type"));
+    assert!(actual.err.is_empty());
+}
+
+#[test]
+fn def_with_specified_list_type_nested() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+        def test-command [ foo: list<list<any>> ] {}
+        "#
+    ));
+
+    assert!(actual.err.is_empty());
+}
```

Comment:
Are those tests redundant to the tests in `src/tests/test_parser.rs` or does it catch another potential area of issues?

Reply:
Yeah I saw that we already had one there. I checked the `src/tests/test_parser.rs` tests and they also check the whole nu binary so there is no issue that it is just a unit test in parser land.